vue-components/src/components/Checkbox.vue

```diff
@@ -0,0 +1,153 @@
+<template>
+	<span class="wikit wikit-checkbox">
+		<input
+			type="checkbox"
+			:id="id"
+			class="wikit-checkbox__input"
+			@change="$emit( 'input', $event.target.checked )"
+			:value="nativeValue"
+			:checked="value"
+			:disabled="disabled"
+		>
+		<label
+			class="wikit-checkbox__label"
+			:for="id"
+		>{{ label }}</label>
+	</span>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import generateUid from '@/components/util/generateUid';
+
+export default Vue.extend( {
+	name: 'Checkbox',
+	props: {
+		value: {
+			required: true,
+			type: Boolean,
+		},
+		nativeValue: {
+			default: null,
+			type: String,
+		},
+		label: {
+			required: true,
+			type: String,
+		},
+		disabled: {
+			type: Boolean,
+			default: false,
+		},
+	},
+	data() {
+		return {
+			// https://github.com/vuejs/vue/issues/5886
+			id: generateUid( 'wikit-Checkbox' ),
+		};
+	},
+} );
+</script>
+
+<style lang="scss">
+$label: '.wikit-checkbox__label';
+
+.wikit-checkbox {
+	&__label {
+		display: flex;
+		align-items: center;
+
+		// TODO: replace with Label mixin
```

Comment:
I assume this is what we were discussing in PR #352 ?

Reply:
yes!